product-web-app/common/req.express.session.mock.ts

```diff
@@ -1,6 +1,6 @@
 const example = {
   isLoggedIn: true, /// to be changed for req.express.session.token
-  role: 'admin',
+  role: '2',
 };
 
 export default example;
```

Comment:
No longer needed as the authorisation system will be implemented soon

Reply:
to be updated after Authorization